doc/visual-programming/source/widgets/classify/adaboost.rst

```diff
@@ -0,0 +1,64 @@
+AdaBoost
+========
+
+.. figure:: icons/adaboost-classification.png
+
+An ensemble meta-algorithm that combines weak learners and adapts to the 'hardness' of each training example.
```

Comment:
Perhaps:

An ensemble meta-algorithm that combines multiple weak learners to build a stronger learning algorithm.